Voidify (#44)
project-trident/trident-website: hugo-site/content/information/features.md

-   **Easy and safe to update:** Project Trident includes utilities to automatically perform system snapshots prior to running any updates. This makes it trivial to perform full-system updates and optionally roll-back right from the bootloader.

-   **No fragmentation:** Project Trident hard drives never need defragmenting and are formatted with OpenZFS, a self-healing filesystem.

-   **Laptop support:** Provides power saving and automatic switching between wired and wifi network connections.
    The rolling release model of Project Trident provides an environment to quickly add support for new hardware.

-   **Easy system administration:** Project Trident provides many graphical tools for performing system administration.

-   **Vibrant community:** Project Trident has a friendly and helpful community.

### Security

The Project Trident system is secure by default.
This section is an overview of the built-in security features.
There are also tips about increasing the security of the installed system beyond the configured defaults.

The security features built into Project Trident include:

> - **All users home directories are encrypted out of box** The individual user directories are only "unlocked" and decrypted when that user logs in to the system. Logging out or rebooting the system automatically locks the home directories again.

Reviewer: ```suggestion
- **All user home directories are encrypted out of box.** The individual user directories are only "unlocked" and decrypted when that user logs in to the system. Logging out or rebooting the system automatically locks home directories.
```

  These are incompatible with the binaries and paths found on a Project Trident system.
  Additional antivirus software is also available in the Appcafe.
  This is useful when sending or forwarding email attachments to users running other operating systems.

- **Potential for serious damage is limited:** Privilege separation between users and the administrator account (root) are built-in.
  Files and directories can only be modified by root or users and groups with permission.
  Any executed programs or scripts are only granted the permissions of that user.
  A malicious program can only infect the files and directories owned by the user.
  Core operating system files are protected.
  Only users that are *wheel* group members can gain administrative access.
  These users are still not allowed to list directory contents or access files outside of the set "user" and "group" permissions.

- **Built-in firewall:** The default firewall ruleset allows Internet access and any available network shares.
  The firewall does not allow any inbound connections to the computer unless authorized by the user.

- **Few default services:** All boot services can be viewed in the Service Manager.

>   Service Manager also allows starting, stopping, and adding or removing from boot any system service.

Reviewer: ```suggestion
  Service Manager also allows starting, stopping, and adding or removing any system service from boot.
```

- **SSH is disabled by default:** SSH can only be enabled by the administrator (root).
  This prevents bots and outside individuals from accessing a Project Trident system.
  If SSH access is required, start the service with the Service Manager or by typing `sudo ln -s /etc/sv/sshd /var/service` in the command line.
  Root access is required.
  A firewall rule opening up TCP port *22* for SSH access is also required. Alternatively, the firewall can be temporarily disabled by running `sudo sv stop nftables` (remember to start it again later, a reboot will also bring the firewall back up).

- **SSH root logins are disabled:** If SSH is enabled, login as a regular user and use `su` or `sudo` for administrative actions.

>   Do not change this setting, as it prevents an unwanted user from having complete access to the system.

Reviewer: ```suggestion
  For your security, do not change this setting. Disabling root logins prevents unwanted users from having complete access to the system.
```
